chpl/chpl-resources/src/main/resources/quartz.properties

```diff
@@ -7,7 +7,7 @@ org.quartz.jobStore.tablePrefix = quartz.QRTZ_
 org.quartz.jobStore.isClustered = true
 
 org.quartz.plugin.jobInitializer.class = org.quartz.plugins.xml.XMLSchedulingDataProcessorPlugin
-org.quartz.plugin.jobInitializer.fileNames = jobs.xml,system-triggers.xml,user-triggers.xml
+org.quartz.plugin.jobInitializer.fileNames = jobs.xml,ocd-3402.xml,system-triggers.xml,user-triggers.xml
```

Comment:
I think we moved towards just putting these jobs & triggers in the jobs.xml and system-triggers.xml files, and we don't need specific ocd-nnnn.xml files. I see the download file generation trigger, but not the cache refresh one